addons/project/static/src/js/tours/project.js

```diff
@@ -121,6 +121,22 @@ registry.category("web_tour.tours").add('project_tour', {
 }, {
     trigger: ".ui-autocomplete > li > a:not(:has(i.fa))",
     auto: true,
+}, {
+    trigger: 'a[name="sub_tasks_page"]',
+    content: 'Open sub-tasks notebook section',
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many .o_list_renderer a[role="button"]',
+    content: 'Add a subtask',
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many div[name="name"] input',
+    content: 'Set subtask name',
+    run: 'text new subtask'
+}, {
+    trigger: 'a[name="sub_tasks_page"]',
+    content: 'Open sub-tasks notebook section',
+    run: 'click',
```

Comment:
This is not a test tour but the tour used for the project onboarding - so I would suggest moving the steps in a test tour or improving the messages a little more, and making them translatable. This applies to the later changes in this file as well. 
To be completely honest I would do both since it would be nice to include the new functionality in the onboarding.

Reply:
I improved the message for the onboarding tour,
i also added the tests in the tests/tours/project_tour.js, but i'm not sure if this is really interesting has this is almost the same flow, tell me what you think thanks for the review :pray: